examples/components/string-reverse/string-reverse.mjs

```diff
@@ -0,0 +1,21 @@
+/**
+ * This module is the JS implementation of the `reverser` WIT world
+ */
+
+/**
+ * This Javascript will be interpreted by `jco` and turned into a
+ * WebAssembly binary with a single export (this `reverse` function).
+ */
+function reverseString(s) {
+  return s.split("")
+    .reverse()
+    .join("");
+}
+
+/**
+ * The Javascript export below represents the export of the `reverse` interface,
+ * which which contains `reverse-string` as it's primary exported function.
+ */
+export const reverse = {
+  reverseString,
+};
```

Comment:
Done! Also I found something that I should probably make an issue for later -- `String.reverse()` didn't work properly actually when I tried to do that instead of `split().reverse().join()`. I'll try to make a testcase later and push it up

Reply:
You might need `.join('')`?

Reply:
I see that is what you have, it seems like it should work.

Reply:
Ah sorry -- a bit late here, but I was saying that *built in* `String#reverse` didn't work. The code I had to write isn't how anyone would normally do it I think, but I wrote it like that only because the builtin didn't work for some reason.